packages/vats/src/core/chain-behaviors.js

```diff
@@ -293,46 +299,73 @@ harden(startTimerService);
  * @param {BootDevices<ChainDevices> & BootstrapSpace & {
  *   consume: { loadCriticalVat: ERef<VatLoader<ChainStorageVat>> }
  * }} powers
+ * @param {{ options?: {
+ *   provisionOnlySmartWallet?: boolean
+ * }}} config
  */
-export const makeBridgeManager = async ({
-  consume: { loadCriticalVat },
-  devices: { bridge },
-  produce: { bridgeManager },
-}) => {
+export const makeBridgeManager = async (
+  {
+    consume: { loadCriticalVat },
+    devices: { bridge },
+    produce: {
+      bridgeManager: bridgeManagerP,
+      provisionBridgeManager,
+      provisionWalletBridgeManager: provisionWalletBridgeManagerP,
+      walletBridgeManager,
+    },
+  },
+  { options: { provisionOnlySmartWallet = true } = {} } = {},
```

Comment:
Where is this set to `false` for `local-chain` and `devnet`?

Reply:
I updated to default to false, and force to true in boot-psm.

Moot however since I ended up removing that approach right after, but good for completeness.